<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunnelDialog</class>
 <widget class="QDialog" name="TunnelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tunnel Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listTunnels">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="grpTunnelProperties">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Tunnel Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>txtName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblHost">
        <property name="text">
         <string>Host</string>
        </property>
        <property name="buddy">
         <cstring>txtHost</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtHost"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblLocalPort">
        <property name="text">
         <string>Local Port</string>
        </property>
        <property name="buddy">
         <cstring>txtLocalPort</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtLocalPort">
        <property name="inputMask">
         <string>00009; </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>txtPort</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPort">
        <property name="inputMask">
         <string>00009; </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblPassword">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblCommand">
        <property name="text">
         <string>Command</string>
        </property>
        <property name="buddy">
         <cstring>txtCommand</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtCommand"/>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkCloseOnTerm">
        <property name="text">
         <string>Automatically close on command termination</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddTunnel">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveTunnel">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listTunnels</tabstop>
  <tabstop>btnAddTunnel</tabstop>
  <tabstop>btnRemoveTunnel</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TunnelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TunnelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
